ansible/roles/ocp-workload-integreatly/defaults/main.yml

```diff
@@ -4,4 +4,10 @@ become_override: True
 silent: False
 install_dir: /tmp/integreatly
 inventory_hosts_file: inventories/hosts
-release_tag: release-0.0.1
\ No newline at end of file
+release_tag: release-1.0.0
+webapp_namespace: webapp
+openshift_master_config_path: /etc/origin/master/master-config.yaml
+admin_username: admin@example.com
+admin_password: Password1
+evals_username: evals@example.com
+evals_password: Password1
```

Comment:
can we avoid to hardcode the password ?
You can set it by default to another password if there is any, like `student_password` ?
Or leave it undefined, and we have to pass it to the playbook run

Reply:
Happy to leave out passwords for now, these credentials are referenced in our public README so are not sensitive. All the same I agree that we might be better off not specifying them.